AWGL/SomaticAmplicon
# Edit thyroid bed files for CRM

# v1.7.10 verification

## Change description

### Change description

Add new panel to match the Pan cancer Thyroid panel, minus any regions not included in CRM. This involved making a new bed file and updating the version of VirtualHood (https://github.com/AWGL/VirtualHood/releases/tag/v1.2.1)

### Related issues

#27 

### Justification for minor update

Doesn’t affect sensitivity – changes are only in the reporting steps.

## Testing description

### Testing description

Clinical scientists:
- check bed file regions are as required
- check that generated analysis sheet is as required

Bioinformatics:
- Verify that new panel/ analysis sheets are applied correctly
- Verify that changes don't affect sensitivity

### Test data location

gen01: `/data/results/validations/SomaticAmplicon-1.7.10`

## Verification results

### Clinical scientist checks

> TODO

### Bioinformatics checks

The analysis sheets and the output in the `hotspot_variants` and `hotspot_coverage` folders were checked manually and only contained variants/gaps from within the new Thyroid bed file regions.

The `*_VariantReport.txt` files for each sample were compared using the unix diff command, all of the variants called were identical for all samples. This covers the entire CRM capture region. The output from this is in `/data/results/validations/SomaticAmplicon-1.7.10/200110_M00766_0288_000000000-CP3DV_v1.7.10/diff`
